cypress/support/pages/admin/pageAdministration.js

```diff
@@ -269,5 +279,68 @@ class Management {
       .find('input')
       .click({ force: true })
   }
+
+  schoolNumberIsAdded() {
```

Comment:
here we follow the method name structure -> verb + noun
so this method name can be like -> addedSchoolNumber()